src/schemathesis/models.py

```diff
@@ -177,6 +178,23 @@ def call_wsgi(self, app: Any = None, headers: Optional[Dict[str, str]] = None, *
         with cookie_handler(client, self.cookies):
             return client.open(**data, **kwargs)
 
+    def call_asgi(
+        self,
+        app: Any = None,
+        base_url: Optional[str] = "http://testserver",
```

Comment:
Why is the default value `http://testserver` ? I'd rather keep it `None` for consistency with the `call` behavior unless it has a special meaning for ASGI apps

Reply:
Hm, it is indeed the default value for `starlette` - https://github.com/encode/starlette/blob/master/starlette/testclient.py#L369